### ISSUE TITLE
Question about Learning curves of Tacotron-2

### ISSUE THREAD

**Issue by dathudeptrai** [espnet/espnet#1914]
Can i ask about learning curves (tensorboard) of Tacotron-2 with location-sensitive attention and guided attention on LJSpeech dataset ?. I'm training it on tensorflow version with mean-var normalization as this repo did, i just want to check validation loss and a behavior of the model. @kan-bayashi

**Comment by kan-bayashi** [comment]
Hi @dathudeptrai. This is the training curve of tacotron2.v3 with phoneme inputs on LJSpeech dataset.
<img width="939" alt="スクリーンショット 2020-05-11 午後2 20 09" src="https://user-images.githubusercontent.com/22779813/81526442-930f8e80-9392-11ea-9499-629d2021fc48.png">
Especially, BCE loss tends to be overfitting in the early stage.

**Comment by dathudeptrai** [comment]
so, l1_loss = mel_before_loss + mel_after+loss ?

**Comment by kan-bayashi** [comment]
Right.
https://github.com/espnet/espnet/blob/1b2782056bc36e53d91f0fec8fa45889244953de/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py#L228

**Comment by dathudeptrai** [comment]
![tensorflow-tacotron](https://user-images.githubusercontent.com/43868663/81526893-1c1ec980-9383-11ea-92ae-e097ff0430a4.png)

**Comment by kan-bayashi** [comment]
No. I did not use scheduled sampling but used reduction factor = 1.

**Comment by dathudeptrai** [comment]
i'm wondering if ur model have some problem when inference w/o teacher forcing mode. Somehow ur model can adapt with mel_pred ? , my model cann't run correctly w/o teacher forcing, it cann't addapt because the training use only prev mel_gt as input of decoder.

**Comment by kan-bayashi** [comment]
I'm not sure how much different the amplitude in your model, but in our case, the generated audio volume is not so changed.
You can listen to our samples: 
https://drive.google.com/open?id=1JFNZapygWsHiP2CXMjTraLzf98h-tEBF
But I've never compared with the model with scheduled sampling.

Is the volume clearly different?

**Comment by dathudeptrai** [comment]
in theory, scheduled teacher forcing should better than teacher forcing (How much better depends on the dataset.). In my vietnamese Dataset, My colleagues train tacotron-2 with mozilla code around 100k steps, r = 1. A trained model can't even inference w/o teacher forcing mode, let say the audio generated very hard to reconigze (a alignment still fine :)) still diagonal :v).

A volume cleary different in my case, i think some people have this problem #1878 .

**Comment by kan-bayashi** [comment]
I see. Thank you for sharing your experiments.
Maybe it depends on not only the language and but also the recording quality.
I will consider introducing the scheduled sampling.

**Comment by dathudeptrai** [comment]
BTW, This repo can be better than its current one :). Honestly, it should transformer on speech (maybe you know transformer on NLP, https://github.com/huggingface/transformers). Maybe you need take some time to refactor overall repo. Just personal opinion: D.
